api/models.py

```diff
@@ -162,6 +162,12 @@ class Node(DatabaseModel):
     name: str = Field(
         description='Name of the node object'
     )
+    path: List[str] = Field(
+        description='Full path with node names from the top-level node'
+    )
+    group: Optional[str] = Field(
```

Comment:
I see the `group` field is `optional` here.
Shouldn't every test belong to one of the test groups?

Reply:
No, the checkout and tarball nodes aren't part of any group.